tools/check_format.py

```diff
@@ -126,6 +126,9 @@ def checkFileContents(file_path):
       printLineError(file_path, line_number, "over-enthusiastic spaces")
     if hasInvalidAngleBracketDirectory(line):
       printLineError(file_path, line_number, "envoy includes should not have angle brackets")
+    for error in PROTOBUF_TYPE_ERRORS:
+      if error in line:
+        printLineError(file_path, line_number, "incorrect protobuf type reference %s" % error)
```

Comment:
can you put the replacement into the error?  That's would make it very actionable :)

Reply:
Sure, though the printout at end says to run `tools/check_format.py fix`, which will fix these automatically.